crates/fluvio-cli/src/topic/delete.rs

```diff
@@ -6,24 +6,35 @@
 
 use tracing::debug;
 use clap::Parser;
-
 use fluvio::Fluvio;
 use fluvio::metadata::topic::TopicSpec;
 use crate::Result;
 
 #[derive(Debug, Parser)]
 pub struct DeleteTopicOpt {
-    /// The name of the Topic to delete
-    #[clap(value_name = "name")]
-    topic: String,
+    /// ignore delete errors if any
+    #[clap(short, long, action, required = false)]
+    ignore_error: bool,
```

Comment:
Not sure if we should do `ignore-error` or if we instead do something like `continue-on-errors` and then at the end we finish with  a non-zero exit code  if there was an error.